tests/unit_tests/test_ecephys.py

```diff
@@ -32,3 +40,77 @@ def test_check_negative_spike_times_some_negative():
         object_name="Units",
         location="/",
     )
+
+
+class TestCheckElectricalSeries(TestCase):
+    def setUp(self):
```

Comment:
```suggestion
    @classmethod
    def setUpClass(cls):
```
Suggest using `setUpClass` any time you make multiple usage of the attributes set at setup; individual test function usage is exactly the same but then the setup only needs to be run once in total (makes CI much faster).

Reply:
@CodyCBakerPhD this actually won't work here. I need a fresh NWBFile every time otherwise I will be trying to add data of the same name to a single NWBFile.